bin/plugin/lib/git.js

```diff
@@ -82,10 +82,20 @@ async function createLocalTag( gitWorkingDirectoryPath, tagName ) {
  *
  * @param {string} gitWorkingDirectoryPath Local repository path.
  * @param {string} branchName Branch Name
+ * @param {Array.<string>} options accepted by git-push
  */
-async function pushBranchToOrigin( gitWorkingDirectoryPath, branchName ) {
+async function pushBranchToOrigin(
+	gitWorkingDirectoryPath,
+	branchName,
+	options = []
+) {
 	const simpleGit = SimpleGit( gitWorkingDirectoryPath );
-	await simpleGit.push( 'origin', branchName );
+
+	// TODO The type definitions for simple-git are wrong and will complain here.
```

Comment:
As the comment states, there seems to be a mismatch between the TS typing declarations and the js version of simple-git. I tried passing the `options` array as the last arg as the type docs say, but it didn't work. 

After inspecting the actual code, I realized thet last param is called "then" and is unused. By checking the implementation, I saw that the right way to pass these options was to pass them as the first argument.

I'll look into it again tomorrow, but if anyone else more acquainted with the codebase has any insights, please let me know :)

Reply:
I managed to fix this by passing the options as an object instead of an array (as the last argument). The interface here is not the most intuitive ¯\_(ツ)_/¯